### ISSUE TITLE
Custom Tooltips Not Working

### ISSUE THREAD

**Issue by danhooper** [m0t0r/ngx-slider#7]
I cannot seem to get custom tooltips with a tooltip template to work.

See https://stackblitz.com/edit/ngx-slider-custom-tooltip

**Comment by graphical-iain** [comment]
Works just fine. Your div just doesn't take up any space.
I just added a `&nbsp;` inside your tooltip div and it works as expected.
![image](https://user-images.githubusercontent.com/6206408/92276980-05ffc080-eec0-11ea-8684-781fd371a61e.png)

https://stackblitz.com/edit/ngx-slider-custom-tooltip-yhqi8d?file=src%2Fapp%2Fapp.component.html

**Comment by danhooper** [comment]
You're right. Thanks for your comment! I think the documentation could be clearer but that works!
